### DISCUSSION TITLE
Fooocus is not installed

### DISCUSSION THREAD

**Discussion by bsdesigner** [LykosAI/StabilityMatrix#718] [General]
After reinstalling the operating system (Windows 11) and reinstalling StabilityMatrix v2.11.2, the Fooocus package is not installed:

![Знімок екрана 2024-06-24 205232](https://github.com/LykosAI/StabilityMatrix/assets/36730003/e296289f-b8e5-4da2-896c-da74d064d167)

After failing to install Fooocus, there was an option in the Packages section to import it:

![Знімок екрана 2024-06-24 233418](https://github.com/LykosAI/StabilityMatrix/assets/36730003/35c2ae99-1507-43b6-b602-881f1e73d0ff)

After I imported Fooocus the generator became available:

![Знімок екрана 2024-06-24 233453](https://github.com/LykosAI/StabilityMatrix/assets/36730003/bfd72657-209e-4269-844f-930ece8371bd)

![Знімок екрана 2024-06-24 233608](https://github.com/LykosAI/StabilityMatrix/assets/36730003/536c79eb-a39b-4c80-a439-f8a8075e1ecb)

But unfortunately, the following error occurred when trying to run it:

![Знімок екрана 2024-06-24 233637](https://github.com/LykosAI/StabilityMatrix/assets/36730003/687268c6-659a-4aa3-920b-684f65f69ee2)

**Comment by monstamash77** [comment]
Me too. Same error after issues with PC, removed Stability Matrix and started again. Tried installing pip 24.0, torchsde 0.2.5, older versions of fooocus, older versions of stability matrix, but always same error

**Comment by bsdesigner** [comment]
After installing the new version of Stability Matrix v2.11.3, all errors disappeared and everything installed perfectly. 
Many thanks to Lykos AI!
![Знімок екрана 2024-06-25 135140](https://github.com/LykosAI/StabilityMatrix/assets/36730003/6f94bdee-c2b7-4412-b4e8-4104369bab8e)
